web_client/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

Comment:
I've found that specifying both `push` and `pull_request` here will run the tests twice if you push to a PR. Just `push` will suffice.

Reply:
Will do!